server/service/src/sync/test/integration/remote/mod.rs

```diff
@@ -0,0 +1,75 @@
+pub(crate) mod invoice;
+pub(crate) mod location;
+pub(crate) mod number;
+pub(crate) mod requisition;
+pub(crate) mod stock_line;
+pub(crate) mod stocktake;
+mod test;
+
+use crate::sync::test::{
+    check_records_against_database,
+    integration::{
+        central_server_configurations::{ConfigureCentralServer, CreateSyncSiteResult},
+        init_db, random_timeout,
+    },
+};
+
+use super::SyncRecordTester;
+
+/// For each test step:
+/// Upsert data to database
+/// Push changes to central server
+/// Reinitilises from cenral server with a fresh database
+/// Check that pulled data matches previously upserted data
+async fn test_remote_sync_record(identifier: &str, tester: &dyn SyncRecordTester) {
+    // util::init_logger(util::LogLevel::Info);
+    random_timeout().await;
```

Comment:
why random_timeout?

Reply:
random_sleep?

Reply:
I changed this once again upstream, now the central data upsert (site creation etc..) use this one:

https://github.com/openmsupply/open-msupply/blob/9c8997df3c00b2227cee8c82c14d7b9cde3df3c9/server/service/src/sync/test/integration/mod.rs#L53-L85

Good point, should have been random_sleep if i kept it